Styles/simple-grid.css

```diff
@@ -0,0 +1,20 @@
+.grid-two-col{
+    display: grid;
+    gap: 2rem;
+    grid-template-columns:  repeat(2 , 1fr);
+    width: 100%;
+    
+}
+
+.col-1 , .col-2 , .col-3{
```

Comment:
If you want the text in the centre then add text-align: centre it will look good.